sql/data_accounting.sql

```diff
@@ -4,6 +4,7 @@
 
 -- Page verification table
 CREATE TABLE IF NOT EXISTS /*$wgDBprefix*/revision_verification (
+	-- Add new fields to DataAccounting\Verification\VerificationLookup
```

Comment:
Not explicit enough. Should be something like "Attention: if you add new columns to this table, make sure to also update the select in path/to/VerificationLookup.php inside the method xxx".

Reply:
i updated the doc